custom_components/bhyve/binary_sensor.py

```diff
@@ -89,3 +90,41 @@ def _on_ws_data(self, data):
 
     def _should_handle_event(self, event_name, data):
         return event_name in [EVENT_FS_ALARM]
+
+    
+class BHyveTemperatureAlert(BHyveDeviceEntity):
```

Comment:
This should be renamed:

```suggestion
class BHyveTemperatureBinarySensor(BHyveDeviceEntity):
```